### PR TITLE
ADD: Textures

### PR DESCRIPTION
**Pull request by branchwelder**

### CODE REVIEW

File: src/La Trop/view/square.cpp [hunk 1]
```diff
@@ -9,18 +9,49 @@
 #include <OpenGL/gl.h>
 #include <OpenGL/glu.h>
 #include <GLUT/glut.h>
+#include <SOIL/SOIL.h>
+
```
Comment (segerphilip) [inline src/La Trop/view/square.cpp:13]: extra space

File: src/La Trop/view/player.cpp [hunk 2]
```diff
@@ -9,21 +9,49 @@
...
 
-    // TODO: Replace this with a texture.
-    glColor3f(0.0, 1.0, 1.0);
+void PlayerView::render(){
+    // Enable 2D textures
+    glEnable(GL_TEXTURE_2D);
+    
+    // Bind texture to the texture target (GL_TEXTURE_2D)
+    glBindTexture(GL_TEXTURE_2D, _tex);
+
```
Comment (segerphilip) [inline src/La Trop/view/player.cpp:33]: extra space

File: src/La Trop/main.cpp [hunk 3]
```diff
@@ -42,11 +42,12 @@ Model *model;
 View *view;
 Controller *controller;
 
+
```
Comment (segerphilip) [inline src/La Trop/main.cpp:45]: extra space

File: src/La Trop/main.cpp [hunk 4]
```diff
@@ -94,6 +95,8 @@ int main(int argc, char **argv) {
     glutSpecialFunc(specialKeys);
     glutReshapeFunc(reshape);
     glutIdleFunc(idleFunc);
+
```
Comment (segerphilip) [inline src/La Trop/main.cpp:98]: extra space

File: src/La Trop/main.cpp [hunk 5]
```diff
@@ -104,6 +107,7 @@ int main(int argc, char **argv) {
     
     // Initialize player that can be moved by the keyboard
     model->initPlayer(15,15, Player(15,15,100));
+
```
Comment (segerphilip) [inline src/La Trop/main.cpp:110]: extra space

File: src/La Trop/view/square.cpp [hunk 6]
```diff
@@ -9,18 +9,49 @@
...
+     );
+}
 
+void SquareView::render(){
+    // Enable 2D textures
+    glEnable(GL_TEXTURE_2D);
+    
+    // Bind texture to the texture target (GL_TEXTURE_2D)
+    glBindTexture(GL_TEXTURE_2D, _tex);
+
```
Comment (segerphilip) [inline src/La Trop/view/square.cpp:33]: extra space

**Review by segerphilip — APPROVED** [review]
We should get a linter. Wait, is there even a linter for Xcode? Whatever. Space. Looks good otherwise.

**Review by sammyers — CHANGES_REQUESTED** [review]
I think we should probably ignore the build folder?

**Review by sammyers — APPROVED** [review]

**Pull request merged**